core/src/main/scala/com/metamx/tranquility/beam/HttpBeam.scala

```diff
@@ -154,13 +157,13 @@ class HttpBeam[A: Timestamper](
                 "eventCount" -> events.size
               )
             )
-            Future.value(0)
+            Future.value(BitSet.empty)
```

Comment:
(outside scope of this PR) This assumes the the consumer either 

1. Doesn't care about duplicates
2. Has some sort of commit magic that prevents it from committing the batch until the events are reported as reported successfully (say that five times fast!)

Is that assumption documented anywhere?